.github/dependabot.yml

```diff
@@ -0,0 +1,27 @@
+version: 2
+
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      # 11 am PST
+      time: "11:00"
+      # PST timezone
+      timezone: "America/Los_Angeles"
+    rebase-strategy: "auto"
+    commit-message:
```

Comment:
```suggestion
    open-pull-requests-limit: 20
    commit-message:
```

Reply:
In case you don't want to be spammed with these PRs, feel free to change the number.

Reply:
Good call, I'll definitely add it in, for now I think 10 will do. I saw that 5 is the default so I think having a little more room for open pull requests would definitely be beneficial. Thanks Denis.